web/templates/pages/members.html

```diff
@@ -10,6 +10,11 @@
 			<h1>Who is who?</h1>
 	<div class="row">
 		<div>
+
+	{% for member in members_list %}
+			<h3>{{member.fullname}}</h3>
+	{% endfor %}
```

Comment:
Yay, dynamic member listings! :tada: The idea is that this will replace the static list of members on the page, so perhaps you can try making the user listing the same as the static list (each name should link to the member's GitHub page and include a profile photo) and grouped by activity level, so we can finally delete the manually created list of members?